app.rb

```diff
@@ -0,0 +1,159 @@
+require './student'
+require './teacher'
+require './book'
+require './rental'
+class App
+  def initialize
+    @books = []
+    @persons = []
+    @rentals = []
+  end
+
+  def run
+    loop do
+      puts 'Please choose an option by entering a number:'
+      puts '1. list all books'
+      puts '2. list all people'
+      puts '3. create a person (teacher or student)'
+      puts '4. create a book'
+      puts '5. create a rental'
+      puts '6. list all rentals for a given person id'
+      puts '7. Exit'
+      option = gets.chomp.to_i
+      process_input(option)
+    end
+  end
+
+  # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
[OPTIONAL] Pull Request should have a descriptive title telling others what changes are inside. It would be better to update that so your project is documented professionally.